docs/townhalls.md

```diff
@@ -17,35 +17,35 @@ You can join with this link, or see the full invite below. https://linkedin.zoom
   <summary>Full Invite</summary>
 
 ```
-John Plaisted is inviting you to a scheduled Zoom meeting.
+Nagarjuna Kanamarlapudi is inviting you to a scheduled Zoom meeting.
 
 Topic: DataHub Town Hall
-Time: Oct 30, 2020 09:00 AM Pacific Time (US and Canada)
+Time: Nov 6, 2020 09:00 AM Pacific Time (US and Canada)
 
 Join Zoom Meeting
-https://linkedin.zoom.us/j/95617940722
+https://linkedin.zoom.us/j/4698262890
```

Comment:
Why changing the ID?

Reply:
I am unable to change the meeting time invite of John's meeting.

Reply:
SGTM